code/modules/mob/language/animal.dm

```diff
@@ -0,0 +1,32 @@
+/decl/language/animal
+	name = "Animal Noises" // translate them!
+	desc = "Some varieties of animal can communicate amongst themselves, apparently."
+	colour = "say_quote"
+	key = "a"
+	shorthand = "A"
+	hidden_from_codex = TRUE
+
+/decl/language/animal/can_be_understood_by(var/mob/living/speaker, var/mob/living/listener)
+	if(!istype(listener))
+		return FALSE
```

Comment:
Possibly this should be `TRUE` instead? Do non-living mobs like ghosts and various special mobs get hit by this proc?